internal/provider/resource_time_static_test.go

```diff
@@ -67,8 +66,7 @@ func TestAccTimeStatic_Triggers(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceName, "triggers.%", "1"),
 					resource.TestCheckResourceAttr(resourceName, "triggers.key1", "value1updated"),
-					testExtractResourceAttr(resourceName, "rfc3339", &time2),
-					testCheckAttributeValuesDiffer(&time1, &time2),
```

Comment:
This test was previously verifying whether `triggers` updates ensured the resource was recreated (by expecting a difference in `rfc3339` attribute value). Is this being tested a different way?